src/Breweries.php

```diff
@@ -4,20 +4,38 @@
 
 namespace Interns2022B;
 
+use Illuminate\Support\Collection;
+use Interns2022B\Models\Brewery;
+
 class Breweries
 {
     public const SEARCH = "choice";
 
-    public function getBreweries(array $breweriesData): array
+    public function getBreweries(Collection $breweriesFactory): Collection
```

Comment:
I would like to rename breweriesFactory to breweries
Let's add doc above method name: `/** @var Collection<Brewery> $breweries */`